source/development/packaging/generate-indexer-package.rst

```diff
@@ -0,0 +1,309 @@
+Wazuh Indexer packages generation guide
+=======================================
+
+General Packaging pre-requisistes:
+==================================
```

Comment:
These 2 sentences are interpreted as a title and merged together, causing rendering issues.

![image](https://github.com/wazuh/wazuh-documentation/assets/15186973/4d8b2a32-19e6-45a6-b065-89d9abf3adb3)

![image](https://github.com/wazuh/wazuh-documentation/assets/15186973/3247eabc-69be-40c9-83e4-f1820cc0e0da)